### ISSUE TITLE
Need more information on discarded_notfiication_center_observer

### ISSUE THREAD

**Issue by tariq235** [realm/SwiftLint#1427]
```
NotificationCenter
            .default
            .addObserver(forName: Notification.Name.loggedIn, object: nil, queue: nil) { [weak self] _ in
                self?.setProfileImage()
                self?.setProfileTitle()
        }
```

Can someone please explain me this warning ?

![screen shot 2017-04-12 at 2 06 35 pm](https://cloud.githubusercontent.com/assets/349211/24943457/5b3c1184-1f89-11e7-809a-d6ba41ccf8c4.png)

**Comment by marcelofabri** [comment]
You need the remove the token you get as a return value from this method. Check [the documentation](https://developer.apple.com/reference/foundation/notificationcenter/1411723-addobserver).
